docker/amd64.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM resin/amd64-debian:jessie
+FROM balenalib/amd64-debian-node
```

Comment:
@stickz , why did you switched to Node stack?

```
$ docker run --rm -it balenalib/amd64-debian-node /bin/sh
Here are a few details about this Docker image (For more information please visit https://www.balena.io/docs/reference/base-images/base-images/): 
Architecture: Intel 64-bit (x86-64) 
OS: Debian Buster 
Variant: run variant 
Default variable(s): UDEV=off 
The following software stack is preinstalled: 
Node.js v15.4.0, Yarn v1.22.4 
Extra features: 
- Easy way to install packages with `install_packages <package-name>` command 
- Run anywhere with cross-build feature  (for ARM only) 
- Keep the container idling with `balena-idle` command 
- Show base image details with `balena-info` command
# 
```

Just asking. I want to refresh the Docker images.

Reply:
I didn't see the `balenalib/amd64-debian` stack. This can be replaced when you refresh the docker images.